client/performance-profiler/components/charts/history-chart.tsx

```diff
@@ -0,0 +1,211 @@
+import { extent as d3Extent, max as d3Max } from 'd3-array';
+import { axisBottom as d3AxisBottom, axisLeft as d3AxisLeft } from 'd3-axis';
+import {
+	scaleLinear as d3ScaleLinear,
+	scaleTime as d3TimeScale,
+	scaleThreshold as d3ScaleThreshold,
+} from 'd3-scale';
+import { select as d3Select, event as d3Event } from 'd3-selection';
+import { line as d3Line, curveMonotoneX as d3MonotoneXCurve } from 'd3-shape';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { timeFormat as d3TimeFormat } from 'd3-time-format';
```

Comment:
Do we want to do this? Shouldn't we import this to use?

Reply:
I added the `d3TimeFormat` and `@types/d3` packages to the project on this branch. I'm not sure if we can just add new dependencies or it needs to be reviewed by someone though.